zebra-rpc/src/methods.rs

```diff
@@ -94,6 +94,20 @@ pub trait Rpc {
     /// Note 3: The `verbosity` parameter is ignored but required in the call.
     #[rpc(name = "getblock")]
     fn get_block(&self, height: Height, verbosity: u8) -> BoxFuture<Result<GetBlock>>;
+
+    /// getbestblockhash
+    ///
+    /// Returns the hash of the current best blockchain tip block.
+    ///
+    /// zcashd reference: <https://zcash.github.io/rpc/getbestblockhash.html>
+    ///
+    /// Result:
+    /// {
+    ///      "hex": String, // Hex encoded block hash
+    /// }
+    ///
```

Comment:
I'm not sure when you wanted to start writing comments in the standard format from #3638.

If you're not going to do it in this PR, can you open a ticket?

Reply:
https://github.com/ZcashFoundation/zebra/issues/3764